src/tag/style.js

```diff
@@ -0,0 +1,59 @@
+import styled, { css } from 'styled-components';
+import Close from '../../assets/close.svg';
+
+const DarkBg = css`
+  background-color: #f0f0f8;
+`;
+
+const HideBorder = css`
+  border: none;
+`;
+
+export const Container = styled.div`
+  text-decoration: none !important;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border-radius: 30px;
+  text-align: center;
+  line-height: 1;
+  padding: ${props => (props.slim ? '6px 12px' : '12px 18px')};
+  background: #fff;
+  border: 1px solid #d1d1e8;
+  font-size: 15px;
+  font-weight: 600;
+  position: relative;
+  bottom: 0;
+  transition: all 0.1s ease-out;
+  margin: 4px;
+  color: #8280b0;
```

Comment:
I think most of the colours in this library are being pulled in via a `colors` file as they should all follow the styleguide of oxygen.
Just double check if it's not added already.